node/src/integration-test/kotlin/net/corda/node/jmx/NodeStatus.kt

```diff
@@ -0,0 +1,45 @@
+package net.corda.node.jmx
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import net.corda.nodeapi.internal.NodeStatus
+import net.corda.testing.driver.DriverParameters
+import net.corda.testing.driver.JmxPolicy
+import net.corda.testing.driver.driver
+import org.junit.Test
+import java.net.HttpURLConnection
+import java.net.HttpURLConnection.HTTP_OK
+import java.net.URL
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class NodeStatus {
```

Comment:
We tend to suffix test classes with `Test`, hence:
```suggestion
class NodeStatusTest {
```